src/components/organisms/CountDown.astro

```diff
@@ -44,15 +50,15 @@ const t = useTranslations(lang);
   </div>
 </div>
 
-<script>
+<script define:vars={{ countdownDate }}>
   // countdown.js
   document.addEventListener("DOMContentLoaded", () => {
     // count-down timer
-    let dest = new Date("apr 25, 2024 10:00:00").getTime();
+    let dest = new Date(countdownDate).getTime();
```

Comment:
What if date parser failed?

Reply:
The cms is responsible of the date validation